fastapi/security/http.py

```diff
@@ -89,8 +89,8 @@ async def __call__(  # type: ignore
                 return None
         try:
             data = b64decode(param).decode("ascii")
-        except (ValueError, UnicodeDecodeError, binascii.Error):
-            raise invalid_user_credentials_exc
```

Comment:
This is behavioural change, why removing the UnicodeDecodeError? That type of exception can be thrown by the `encode()` call.